pkg/controller/verticalpodautoscaler/verticalpodautoscaler_controller.go

```diff
@@ -257,17 +257,17 @@ func (r *Reconciler) Reconcile(request reconcile.Request) (reconcile.Result, err
 			klog.Info(msg)
 			continue
 		}
-		if err := r.UpdateAutoscaler(vpa, params); err != nil {
+		if err, updated := r.UpdateAutoscaler(vpa, params); err != nil {
```

Comment:
let's reverse the order here of the returned error, usually golang does the error last, (updated, err).

Reply:
Thanks. Updated.